hub/integrations/tests/test_pytorch.py

```diff
@@ -68,18 +68,19 @@ def test_pytorch_small_old(local_ds):
 def test_pytorch_large_old(local_ds):
     import torch
 
-    local_ds.create_tensor("image")
+    local_ds.create_tensor("image", dtype="int64")
     arr = np.array(
         [
             np.ones((4096, 4096)),
             2 * np.ones((4096, 4096)),
             3 * np.ones((4096, 4096)),
             4 * np.ones((4096, 4096)),
-        ]
+        ],
+        dtype="int64",
     )
     local_ds.image.extend(arr)
-    local_ds.create_tensor("classlabel")
-    local_ds.classlabel.extend(np.array([i for i in range(10)]))
+    local_ds.create_tensor("classlabel", htype="class_label")
+    local_ds.classlabel.extend(np.array([i for i in range(10)], dtype="uint32"))
```

Comment:
Why not specify this in `create_tensor`?

Reply:
`class_label` is predefined with `uint32`

Reply:
OH I missed that this was a param on the numpy array and not the extend method